fastestimator/dataset/dataloader.py

```diff
@@ -111,15 +111,14 @@ def __init__(self,
         # We could disable pre-collating when num_workers=0, but this would lead to inconsistent batch ordering between
         # single- and multi-processing.
 
-        super().__init__(dataset=dataset,
-                         batch_size=batch_size,
-                         sampler=sampler,
-                         num_workers=num_workers,
-                         persistent_workers=False,
-                         collate_fn=functools.partial(_pre_collate,
-                                                      try_fn=self.fe_collate_fn,
-                                                      postprocess_fn=postprocess_fn),
-                         worker_init_fn=lambda _: np.random.seed(random.randint(0, 2 ** 32 - 1)))
+        super().__init__(
+            dataset=dataset,
```

Comment:
I don't think this formatting adjustment was necessary, now that you've got the IDE configured can you roll back these lines and see if it still changes their formatting?

Reply:
it still changes the format. i think its trying to keep the collate_fn in single line instead of 3.